### PR TITLE
Tidy/Restructure Account API description

### PR DESCRIPTION
**Pull request by lornajane**
# Description

Restructure the Account API to have named request bodies along the same lines as our other APIs (and to please the tool I'm using)

# Fixes

* Improves structure of Account API description

# Checklist

- [x] version number incremented (in the `info` section of the spec)

### CODE REVIEW

File: definitions/account.yml [hunk 1]
```diff
@@ -630,6 +590,56 @@ components:
           description: Internal Trace ID
           example: bf0ca0bf927b3b52e3cb03217e1a1ddf
 
+    accountSettingsRequest:
+      type: object
+      properties:
+        moCallBackUrl:
+          description: >-
+            The URL where Vonage will send a webhook when an SMS is received
```
Comment (marklewin) [inline definitions/account.yml:598]: I always wonder where this "sending a webhook" terminology comes from. Vonage doesn't "send" a webhook, it makes a request to the webhook?

**Review by marklewin — COMMENTED** [review]
Looks good @lornajane - just one comment on the language (which I know isn't strictly the reason for the PR, but it would make sense to fix it here if we are in agreement?)

File: definitions/account.yml (hunk 1 above)
Comment (lornajane) [reply definitions/account.yml:598]: This isn't bothering me at all, you can definitely think of it as "sending a payload to a webhook endpoint". If we have standard wording I am of course happy to conform (or to have you fix that directly on this PR if you'd like)

Comment (marklewin) [reply definitions/account.yml]: @lornajane Changed some of the wording around webhooks. Also, my understanding is that you cannot set a number-specific DLR webhook, so attempted to make that clear in the docs (`$accountSettingsRequest`)

Comment (lornajane) [reply definitions/account.yml]: Oh this is good, thanks for taking the care to make this better!

**Comment by lornajane** [comment]
I like your changes @marklewin but I can't approve my own PR so consider your additions approved, and I've re-requested your review

**Review by marklewin — APPROVED** [review]
Approving Lorna's approval of my changes :)

**Pull request merged**